hadoop-ozone/dist/src/main/compose/ozone/docker-config

```diff
@@ -51,6 +51,7 @@ OZONE-SITE.XML_hdds.scm.replication.thread.interval=15s
 OZONE-SITE.XML_hdds.scm.replication.under.replicated.interval=5s
 OZONE-SITE.XML_hdds.scm.replication.over.replicated.interval=5s
 OZONE-SITE.XML_hdds.scm.wait.time.after.safemode.exit=30s
+OZONE-SITE.XML_ozone.scm.pipeline.creation.auto.factor.one=false
```

Comment:
Why do we need to disable "factor one" pipeline auto creation?

Reply:
problem is that RATIS/ONE pipelines are auto created. when the pipeline create command is run , it won't create any more ONE pipelines if they are already present throwing an error. 
Another solution rather than disabling this flag is to set the PIPELINE to any one of the ONE pipelines in the list.